webapp/pages/post/_id/_slug/index.spec.js

```diff
@@ -31,7 +31,7 @@ describe('PostSlug', () => {
       $filters: {
         truncate: a => a,
       },
-      // If you mocking router, than don't use VueRouter with lacalVue: https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html
+      // If you are mocking the router, then don't use VueRouter with localVue: https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-05T14:50:18Z" title="Monday, August 5th 2019, 4:50:18 pm +02:00">Aug 5, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -31,7 +31,7 @@ describe('PostSlug', () => {
       $filters: {
         truncate: a => a,
       },
-      // If you mocking router, than don't use VueRouter with lacalVue: https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html
+      // If you are mocking the router, then don't use VueRouter with localVue: https://vue-test-utils.vuejs.org/guides/using-with-vue-router.html
```


That’s an unrelated file !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
😂

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-06T20:31:23Z" title="Tuesday, August 6th 2019, 10:31:23 pm +02:00">Aug 6, 2019</time>_

---


!!!111eleven :angry:

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-08-07T04:55:03Z" title="Wednesday, August 7th 2019, 6:55:03 am +02:00">Aug 7, 2019</time>_

---


what is this supposed to mean? hahaha

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-08T21:21:07Z" title="Thursday, August 8th 2019, 11:21:07 pm +02:00">Aug 8, 2019</time>_

---


When people get really angry, they use a lot of exclamation marks and don't manage to keep the shift key pressed!!!11